app/population/view/daily.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
```

Comment:
It doesn't look like you're using this anymore but it's also already defined globally so you shouldn't need to define it again.

Reply:
Removed unneeded global imports